### ISSUE TITLE
Lack of automatic spool directory causes some errors in log files

### ISSUE THREAD

**Issue by qosobrin** [sympa-community/sympa#215]
I am using sympa, version 6.2.16, which I installed from the Debian Stretch package. After renaming a list I noticed the following error in sympa.log:
```
Feb 28 11:42:08 vm021104 wwsympa[24193]: err main::#1662 > main::do_rename_list#13370 > Sympa::Admin::rename_list#1033 Unable to open "/var/spool/sympa/automatic" spool: No such file or directory
``` 
which was caused because the directory /var/spool/sympa/automatic (related to  the queueautomatic variable) was missing. I went on and created it manually but, afterwards, I realized that some other spool directories had been created on demand previously:
```
Feb 23 17:25:23 vm021104 wwsympa[11345]: info Sympa::Spool::_create() Creating directory /var/spool/sympa/viewmail of Sympa::Spool::Moderation
Feb 23 17:25:23 vm021104 wwsympa[11345]: info Sympa::Spool::_create() Creating directory /var/spool/sympa/viewmail of Sympa::Spool::Moderation
Feb 23 17:25:23 vm021104 wwsympa[11345]: info Sympa::Spool::_create() Creating directory /var/spool/sympa/viewmail/mod of Sympa::Spool::Moderation
Feb 23 17:25:23 vm021104 wwsympa[11345]: info Sympa::Spool::_create() Creating directory /var/spool/sympa/viewmail/mod of Sympa::Spool::Moderation
Feb 23 17:25:23 vm021104 wwsympa[11345]: info Sympa::Spool::_create() Creating directory /var/spool/sympa/subscribe of Sympa::Spool::Auth
Feb 23 17:25:23 vm021104 wwsympa[11345]: info Sympa::Spool::_create() Creating directory /var/spool/sympa/subscribe of Sympa::Spool::Auth
```
Shouldn't all of them be created on demand? Is there any reason for this different behaviour?

By the way, after manually creating the spool directory I tried to rename another list and the error didn't come up again.

Thank you very much.
Regards.

**Comment by ikedas** [comment]
Hi @qosobrin,

This problem seems fixed on 6.2.23b.1. Please check 6.2.24 or later if possible.

**Comment by qosobrin** [comment]
Ok, thanks. Regards.
